mixer/adapter/dogstatsd/operatorconfig/config.yaml

```diff
@@ -0,0 +1,96 @@
+apiVersion: "config.istio.io/v1alpha2"
+kind: metric
+metadata:
+ name: request_count
```

Comment:
can we reuse the "default" metrics already defined instead of defining new ones (for example: `requestcount.metric`) ? These look nearly identical.